content/projects/simple-website/_index.md

```diff
@@ -0,0 +1,71 @@
+
+## Instructions
+
+Create a personal website to showcase your talents and ambitions.
+
+The content and information you choose to display is up to you. There should be some information about your skills, interests, and previous experience, as well as information on how to get in touch with you. Think about the user experience when designing your layout and navigation.
+
+The HTML and CSS should be developed in a logical way with good naming conventions that would allow another developer to easily understand and build upon your code. There is something called "Semantic Markup" - read all about it [here](https://seekbrevity.com/semantic-markup-important-web-design/).
+
+You can also make use of this cool [markup validation tool](https://validator.w3.org/) to validate the HTML code used in your website.
+
+Your web page files and folders should be organized and named in a logical manner.
+
+Once you have finished your design and are satisfied with the style and content, upload the website to be hosted on GitHub Pages.
+
+Please submit a link to your website. DO NOT submit a link to your github repo. We dont want to clone your website, we want to look at what you built.
+
+To learn more about Github Pages visit this link https://guides.github.com/features/pages/
+
+### Requirements:
+
+- Website can be a single page or have multiple pages (the structure is up to you), but it must contain the following information:
+  - About me
+  - Skills (Your tech stack: the languages and tools you use)
+  - Projects/Portfolio (for now this might be a bit empty, but if you've worked on anything cool then tell us about it)
+  - Contact (social media, email, phone, etc)
+- Host your site as a Github Page. In your README file include a link to your github page for easy navigation.
+
+## Some advice
+
+- Your portfolio can be a simple single page portfolio that contains the above mentioned requirements.
+- Make sure that only the necessary information is presented.
+- Make sure your stuff is tidy. Use whitespace as needed. Make sure your pictures are not squashed or stretched.
+- Website must be responsive to different screen sizes - use a mobile-first approach!
+
+### Links
+
+- [How to make a GitHub Page](https://pages.github.com/)
+- [HTML Semantics](https://www.w3schools.com/html/html5_semantic_elements.asp)
+- [More info on CSS Grid](https://css-tricks.com/snippets/css/complete-guide-grid/)
+- [Building a Data Science Portfolio](https://www.dataquest.io/blog/build-a-data-science-portfolio/)
+- [Tips to include towards creating an effective Portfolio](https://www.freecodecamp.org/news/how-to-build-an-awesome-data-science-portfolio/)
+
+
+## Instructions for reviewer 
+
+- The link submitted should direct you to the website.
+- All social media accounts and work profiles should work.
+- Font and text size should be clear.
+- They should make use of the entire web page.
+- It should be responsive to all screen sizes.
```

Comment:
Does responsiveness matter for data scientists? Just check the NCIT standard.

This should just do enough to hit the NCIT standard. Less is more